conllu/models.py

```diff
@@ -19,70 +35,71 @@ class Token(OrderedDict):
         "xpos": "xpostag",
     }
 
-    def get(self, key, *args, **kwargs):
+    def get(self, key: str, default: T.Optional[T.Any] = None) -> T.Any:
         if key not in self and key in self.MAPPING:
             key = self.MAPPING[key]
 
-        return super(Token, self).get(key, *args, **kwargs)
+        return super(Token, self).get(key, default)
 
-    def __missing__(self, key):
+    def __missing__(self, key: str) -> T.Any:
         if key in self.MAPPING:
             return self[self.MAPPING[key]]
 
         raise KeyError("'" + key + "'")
 
-class TokenList(list):
-    metadata = None
+class TokenList(T.List[Token]):
+
+    metadata: T.Optional[_MetadataType] = None
```

Comment:
Is the class level declaration here meant to assign a default value of `None`, or just serve as documentation? If the latter is the case, then simply having `metadata: _MetadataType` would suffice here.

Reply:
Yeah, this could probably be simplified. But since people could use this, changing it would break backwards compatibility, something I'd like to avoid. So let's keep it this way for now.